scripts/01_generate_index_variables.R

```diff
@@ -260,14 +261,13 @@ fb_stats <- fb_vars %>%
 
 # Income Indicator
 # Download in Zip file from HUD website, unzip and rename
-download.file("https://www.huduser.gov/portal/datasets/cp/2012thru2016-140-csv.zip", 
+download.file("https://www.huduser.gov/portal/datasets/cp/2013thru2017-140-csv.zip", 
```

Comment:
The `download.file()` fxn spits out a timeout error for me because the file seems to be big. So you should increase the timeout to a few minutes  (instead of default 1 min). You can do this by including the following line before:

```
options(timeout=180)
````

I would also set `method="curl"` instead of `method="libcurl"` as I've found that to be faster. I will probably make a commit that makes both of these changes for you